packages/toggle-switch/toggle-switch.scss

```diff
@@ -3,207 +3,149 @@
+    --jkl-toggle-switch-knob-border: #{jkl.$color-snohvit};
+    --jkl-toggle-switch-knob-background: #{jkl.$color-granitt};
+    --jkl-toggle-switch-indicator-color: transparent;
+    --jkl-toggle-switch-indicator-color--on: #{jkl.$color-granitt};
+    --jkl-toggle-switch-disabled-color: #{jkl.$color-stein};
+}
 
+.jkl-toggle-switch {
     @include jkl.reset-outline;
+    background: transparent;
+    color: var(--jkl-color);
+    padding: 0;
     cursor: pointer;
-    position: relative;
+    -webkit-tap-highlight-color: transparent;
+
     display: flex;
-    height: var(--jkl-toggle-height);
-    padding: 0;
+    flex-direction: row-reverse;
     align-items: center;
-
-    background-color: transparent;
-    color: currentColor;
+    gap: var(--jkl-spacing-8);
 
     @include jkl.use-font-variables("jkl-toggle");
 
-    &__container {
-        display: flex;
-        flex-direction: column;
-    }
+    --jkl-toggle-switch-knob-position: 0;
 
-    &__help-label {
-        margin-top: jkl.$spacing-s;
+    &[aria-pressed="true"] {
+        --jkl-toggle-switch-background: var(--jkl-toggle-switch-border);
+        --jkl-toggle-switch-knob-border: var(--jkl-toggle-switch-knob-background);
+        --jkl-toggle-switch-indicator-color: var(--jkl-toggle-switch-indicator-color--on);
+        --jkl-toggle-switch-knob-position: calc(
+            var(--jkl-toggle-switch-width) - var(--jkl-toggle-switch-knob-size) - var(--jkl-toggle-switch-padding) * 2
+        );
     }
 
-    &__slider {
-        border-radius: 99rem;
-        display: inline-block;
-        height: var(--jkl-toggle-height);
-        position: relative;
-        width: var(--jkl-toggle-width);
-        margin-right: jkl.$spacing-xs;
-        pointer-events: none;
-
-        /* The line the dot slides on */
-        &::before {
-            content: "";
-            background-color: var(--jkl-toggle-switch-main-slider-color);
-            height: 1px;
-            left: calc(var(--jkl-toggle-line-left-right-margin) * 0.5);
-            position: absolute;
-            top: 50%;
-            width: calc(var(--jkl-toggle-width) - var(--jkl-toggle-line-left-right-margin));
-        }
-
-        /* Sliding dot */
-        &::after {
-            box-shadow: inset 0 0 0 var(--jkl-toggle-dot-white-line-inset) var(--jkl-toggle-switch-main-slider-color),
-                inset 0 0 0 calc(var(--jkl-toggle-dot-white-line-inset) + jkl.rem(2px))
-                    var(--jkl-toggle-switch-main-slider-background-color);
-            content: "";
-            border-radius: 99rem;
-            position: absolute;
-            left: 0;
-            top: 0;
-            width: var(--jkl-toggle-height);
-            height: var(--jkl-toggle-height);
-            background-color: var(--jkl-toggle-switch-main-slider-color);
-            transition: transform 200ms cubic-bezier(0.68, -0.55, 0.265, 1.55);
-        }
+    // TODO: Hvis vi fortsatt skal ha disabled state må den kanskje designes litt bedre
```

Comment:
Er det tegnet noe? Hvis ikke tenker jeg vi kan fjerne den, samme som vi gjorde for kontekstuell meny.